crates/host/src/secrets.rs

```diff
@@ -26,8 +26,9 @@ pub(crate) struct SecretReference {
     /// The version of the secret to retrieve. If not supplied, the latest version will be used.
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub version: Option<String>,
-    /// The policy that defines configuration options for the backend.
-    pub policy_properties: String,
+    /// The policy that defines configuration options for the backend. This is a serialized
+    /// JSON object that will be passed to the backend for policy evaluation.
+    pub policy: String,
```

Comment:
Any reason not to make this a `serde_json::Value`? If we want to require this to be a JSON object, why not make it freeform instead of a string?

Reply:
Good idea